app/src/pages/admin/projectsPage/messages.js

```diff
@@ -50,9 +50,21 @@ export const messages = defineMessages({
     id: 'ProjectDetailsPageMembers.title',
     defaultMessage: 'Members',
   },
-  [`${EVENTS}Title`]: {
+  [`${MONITORING}Title`]: {
```

Comment:
Can we combine these messages with presented below to avoid duplications?

![image](https://github.com/reportportal/service-ui/assets/31861805/8c48cf52-2d28-4056-a2de-d56424e98800)